pkg/ingester/ingester.go

```diff
@@ -627,6 +627,7 @@ func (i *Ingester) PushWithCleanup(ctx context.Context, req *mimirpb.WriteReques
 			otlog.Int("numseries", len(req.Timeseries)))
 	}
 
+	oooAllowance := i.limits.OOOAllowance(userID)
```

Comment:
This value can change in runtime. Here we would reflect latest value, but when creating TSDB we only use given value at the time and don't reflect updates.